src/DataSheet.js

```diff
@@ -349,6 +350,38 @@ export default class DataSheet extends PureComponent {
         }
         return false
       }
+
+      if (this.props.allowNav) {
```

Comment:
"allowNav" seems like a boolean here. 
maybe isCellNavigable(cell) would be a better fit.

Reply:
done